### PR TITLE
Docker disni version

### PR DESCRIPTION
**Pull request by PepperJo**
Automatically get libdisni version from DiSNI jar.
Before we had to manually update the DiSNI version
for building the native library on every release.

### CODE REVIEW

File: docker/RDMA/Dockerfile [hunk 1]
```diff
@@ -17,14 +17,14 @@
 FROM apache/incubator-crail:1.1
 MAINTAINER Apache Crail <dev@crail.apache.org>
 
-# TODO: automate update version
-ARG DISNI_COMMIT="v1.7"
-
 RUN echo "Crail-$LOG_COMMIT install rdma libraries and autotools" && \
```
Comment (asqasq) [inline docker/RDMA/Dockerfile]: Can we remove LOG_COMMIT? It seems not to be initialized?

Comment (PepperJo) [reply docker/RDMA/Dockerfile]: Good question. Let me check where this comes from.

Comment (PepperJo) [reply docker/RDMA/Dockerfile]: Should be initialized by base image (ENV).

Comment (asqasq) [reply docker/RDMA/Dockerfile]: Ah good, then it is fine.

Comment (PepperJo) [reply docker/RDMA/Dockerfile]: Thx. Will merge.

**Pull request merged**